ui/app/AppLayouts/Profile/panels/ProfileSocialLinksPanel.qml

```diff
@@ -0,0 +1,214 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import QtQuick.Shapes 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+import "../popups"
+
+Control {
+    id: root
+
+    property var profileStore
+    property var socialLinksModel
+
+    background: null
+
+    implicitHeight: layout.implicitHeight + linksView.contentHeight
+
+    Component {
+        id: addSocialLinkModalComponent
+        AddSocialLinkModal {
+            onAddLinkRequested: root.profileStore.createLink(linkText, linkUrl, linkType, linkIcon)
+        }
+    }
+
+    Component {
+        id: modifySocialLinkModal
+        ModifySocialLinkModal {
+            onUpdateLinkRequested: root.profileStore.updateLink(uuid, linkText, linkUrl)
+            onRemoveLinkRequested: root.profileStore.removeLink(uuid)
+        }
+    }
+
+    contentItem: ColumnLayout {
+        id: layout
+        spacing: 12
+
+        RowLayout {
+            Layout.fillWidth: true
+            Layout.bottomMargin: Style.current.halfPadding
+            StatusBaseText {
+                text: qsTr("Links")
+                color: Theme.palette.baseColor1
+            }
+            Item { Layout.fillWidth: true }
+            StatusLinkText {
+                text: qsTr("＋ Add more links")
+                color: Theme.palette.primaryColor1
+                font.pixelSize: Theme.tertiaryTextFontSize
+                font.weight: Font.Normal
+                onClicked: Global.openPopup(addSocialLinkModalComponent)
+            }
+        }
+
+        // empty placeholder when no links; dashed rounded rectangle
+        Shape {
```

Comment:
Maybe you can make it a component in a different file. You will need it, at least, in empty communities, assets and so on tabs.. and probably it will be extended also to other views!

Reply:
Yup, good idea!

Reply:
Done